internal/envconfig/envconfig.go

```diff
@@ -59,6 +59,7 @@ var (
 func init() {
 	switch strings.ToLower(os.Getenv(requireHandshakeStr)) {
 	case "on":
+	default:
```

Comment:
Go switch statements don't fall through. This breaks "on" I would think. (spotted by @icio)